test/util/rpcauth-test.py

```diff
@@ -21,6 +21,7 @@ def setUp(self):
         with open(config_path, encoding="utf8") as config_file:
             config.read_file(config_file)
         sys.path.insert(0, os.path.dirname(config['environment']['RPCAUTH']))
+        sys.dont_write_bytecode = True
```

Comment:
Should also update https://github.com/bitcoin/bitcoin/blob/2987ba66375863cdfb0e6065a36c5d3302499c0b/Makefile.am#L332

Reply:
Thanks! Updated.